Source/ACE/Factories/LootGenerationFactory.cs

```diff
@@ -29,44 +30,14 @@ public static void Spawn(WorldObject inventoryItem, Position position)
             LandblockManager.AddObject(inventoryItem);
         }
 
-        public static WorldObject CreateTrainingWand(Player player)
+        public static WorldObject CreateTestWorldObject(Player player, ushort weenieId = 12748)
```

Comment:
Really not a fan of hard-coded IDs like this.  I know we have a weenie enum started for certain test cases - at least add the value there and reference it here.